<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassMan</class>
 <widget class="QMainWindow" name="PassMan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PassMan</string>
  </property>
  <widget class="QWidget" name="passManCentralWidget">
   <widget class="QTableWidget" name="entryTableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>250</width>
      <height>520</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="listLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Entry List:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entryNameLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>290</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="entryNameLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Entry Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>290</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>290</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="repeatedPasswordLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeated Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="repeatedPasswordLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>290</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="notesTextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>290</width>
      <height>130</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="entryNotesLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Entry Notes:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generatePasswordButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Password</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="revealPasswordCheckBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reveal Password</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="passwordStrengthBar">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>535</y>
      <width>215</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="passwordStrengthLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>535</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Strength:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPassMan">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Database"/>
    <addaction name="actionOpen_Database"/>
    <addaction name="actionSave_Database"/>
    <addaction name="actionSaveas_Database"/>
    <addaction name="actionClose_Database"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEntries">
    <property name="title">
     <string>Entries</string>
    </property>
    <addaction name="actionAdd_Entry"/>
    <addaction name="actionCopy_Entry_Username"/>
    <addaction name="actionCopy_Entry_Password"/>
    <addaction name="actionDelete_Entry"/>
    <addaction name="actionAuto_Type_Entry"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPassword_Generator"/>
    <addaction name="actionPassword_Strength_Calculator"/>
    <addaction name="actionYubiKey_Tester"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_Use"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuPassMan"/>
   <addaction name="menuEntries"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionNew_Database">
   <property name="text">
    <string>New Database</string>
   </property>
  </action>
  <action name="actionOpen_Database">
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionSave_Database">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Database</string>
   </property>
  </action>
  <action name="actionSaveas_Database">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Database as</string>
   </property>
  </action>
  <action name="actionHow_to_Use">
   <property name="text">
    <string>How to Use</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPassword_Generator">
   <property name="text">
    <string>Password Generator</string>
   </property>
  </action>
  <action name="actionPassword_Strength_Calculator">
   <property name="text">
    <string>Password Strength Calculator</string>
   </property>
  </action>
  <action name="actionYubiKey_Tester">
   <property name="text">
    <string>YubiKey Tester</string>
   </property>
  </action>
  <action name="actionAdd_Entry">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Entry</string>
   </property>
  </action>
  <action name="actionDelete_Entry">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete Entry</string>
   </property>
  </action>
  <action name="actionAuto_Type_Entry">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Auto-Type Entry</string>
   </property>
  </action>
  <action name="actionClose_Database">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Database</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCopy_Entry_Username">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy Entry Username</string>
   </property>
  </action>
  <action name="actionCopy_Entry_Password">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy Entry Password</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
